shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -52,17 +52,22 @@ namespace {
 
 std::unique_ptr<shared_model::interface::QueryResponse>
 shared_model::proto::ProtoQueryResponseFactory::createAccountAssetResponse(
-    std::vector<std::unique_ptr<shared_model::interface::AccountAsset>> assets,
+    const std::vector<interface::types::AccountIdType> account_ids,
+    const std::vector<interface::types::AssetIdType> asset_ids,
+    const std::vector<shared_model::interface::Amount> balances,
```

Comment:
I don't think these parameters should be const. Purpose of accepting them by-value was allow moving them without copying inside the functions, but now they are getting copied to lambda because of const modifier. So, I think, const should be removed from here and other functions, which were touched

Reply:
Please rework with vector of tuples. Otherwise it is not guaranteed by interface that all vectors have the same size